tools/points-service/readme.md

```diff
@@ -0,0 +1,47 @@
+
+## Vanilla Registry Events
```

Comment:
I think we have added more events no?

This is the pain with having READMEs. Either update it or remove it.